doc/src/user_guide.rst

```diff
@@ -592,10 +617,11 @@ Example::
     quad-deg = 10
     quad-pts = gauss-legendre
 
-[solver-interfaces-tri]
-^^^^^^^^^^^^^^^^^^^^^^^
+[solver-interfaces-tri-{mg-*p*}]
```

Comment:
Is solver-interfaces-tri{-mg-*p*} better? The idea was to show that the whole -mg-*p* is optional, and *p* is replaced by the level you want to set params for?